jobs/bump-lockfile.Jenkinsfile

```diff
@@ -224,7 +224,9 @@ lock(resource: "bump-${params.STREAM}") {
                             // compressed one
                             shwrap("cosa compress --artifact=metal")
                         }
-                        kolaTestIso(cosaDir: env.WORKSPACE, arch: arch, marker: arch)
+                        stage("${arch}:Kola:TestISO") {
```

Comment:
Should it be `${arch}:kola:testiso` for consistency with the `${arch}:kola` and `${arch}:kola:upgrade`?

Reply:
yep. updated.